code/__DEFINES/traits.dm

```diff
@@ -325,6 +325,7 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_NOSOFTCRIT "nosoftcrit"
 #define TRAIT_MINDSHIELD "mindshield"
 #define TRAIT_DISSECTED "dissected"
+#define TRAIT_SURGICALLY_ANALYZED "surgerically_analyzed"
```

Comment:
What's the point in having these 2 be separate defines?
Can't it just be one trait that doesn't go away, and is not blocking autopsy surgery from starting?

Reply:
I suppose so. It might also be good for futureproofing to prevent multiple autopsies on the same corpse, or tracking death and revival in some way.

I might give it a purpose by, say, examining a corpse so that a coroner visually can tell when they're looking at a fresh corpse or not. That could be helpful.

Reply:
I have done that.